backend/protos/xyz/block/ftl/v1/ftl.proto

```diff
@@ -16,6 +16,41 @@ message PingResponse {
   optional string not_ready = 1;
 }
 
+message ModuleContextRequest {
+  string module = 1;
+}
+
+message ModuleContextResponse {
+  message Ref {
+    optional string module = 1;
```

Comment:
I think we probably don't even need this in hindsight. The module isn't going to care where the secret comes from, just the name of it...?

Reply:
Can't there be naming conflicts though?
Module config named X, vs global config named X

Reply:
Not from a module's perspective - all it sees is a flat set of keys

Reply:
Ah, I didn't realise that. that'll simplify some still in the main PR as well